webtau-http/src/main/java/com/twosigma/webtau/http/HttpDataNodes.java

```diff
@@ -70,6 +70,7 @@ private void populateHeaderData() {
         addCamelCaseVersion("Location", "location", asIs);
         addCamelCaseVersion("Content-Location", "contentLocation", asIs);
         addCamelCaseVersion("Content-Length", "contentLength", Integer::valueOf);
+        addCamelCaseVersion("Content-Encoding", "contentEncoding", asIs);
```

Comment:
I will refactor/rename/tidy-up this class in a subsequent PR.  Didn't want to muddle this PR with unrelated changes.